config/component_metadata.yaml

```diff
@@ -0,0 +1,22 @@
+releases:
+  - name: KServe
+    version: v0.14.0
+    repoUrl: https://github.com/kserve/kserve/
+  - name: ModelMesh Serving
+    version: v0.12.0-rc0
```

Comment:
However, IMHO, it is not a good thing to use the `rc` release tag here, users might think that our product does not use the final / stable release from upstream.

Reply:
Agreed. Will update the rc to full release.